### PR TITLE
Feature/fe setup

### PR DESCRIPTION
**Pull request by chris-lawton**
Base setup:

- adds colour hexes
- adds fonts and typography scales
- adds svgs
- adds link styles

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/atoms/sprites/link-anchor/link-anchor.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<a class="link link--anchor" herf="{{value.href}}">
+    {{ value.text }}
+    <svg><use xlink:href="#arrow"></use></svg>
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/atoms/sprites/link-anchor/link-anchor.html]: Think we should be adding height and widths to SVG tags?

File: rca/project_styleguide/templates/patterns/atoms/sprites/link-primary/link-primary.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<a class="link link--primary" href="{{value.href}}">
+    {{ value.text }}
+    <svg><use xlink:href="#arrow"></use></svg>
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/atoms/sprites/link-primary/link-primary.html]: Same here (width and height tag)

File: rca/project_styleguide/templates/patterns/atoms/sprites/link-secondary/link-secondary.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<a class="link link--secondary" href="{{value.href}}">
+    {{ value.text }}
+    <svg><use xlink:href="#arrow"></use></svg>
```
Comment (nicklee) [inline rca/project_styleguide/templates/patterns/atoms/sprites/link-secondary/link-secondary.html]: Same here (width and height tag)

File: rca/project_styleguide/templates/patterns/atoms/sprites/link-anchor/link-anchor.html (hunk 1 above)
Comment (chris-lawton) [reply rca/project_styleguide/templates/patterns/atoms/sprites/link-anchor/link-anchor.html]: I've set them via the stylesheet although I think either way is fine.

**Pull request merged**